### ISSUE TITLE
post request with form-data returns empty request.body

### ISSUE THREAD

**Issue by justinventuri** [iron-meteor/iron-router#1020]
![screen shot 2014-11-06 at 4 02 36 pm](https://cloud.githubusercontent.com/assets/70282/4943858/4d2c5bbe-65f8-11e4-8ada-55cd8745edfe.png)

``` coffee
 @route '/api/content_upload',
    where: 'server'
    action: ->
      request = @request
      response = @response
      response.writeHead 200, {'Content-type': 'application/json; charset=utf-8' }
      response.end JSON.stringify( {ok: 'cool'} )
```

**Comment by tagrudev** [comment]
This is still relevant

**Comment by DirkStevens** [comment]
@tagrudev - this is a duplicate of https://github.com/iron-meteor/iron-router/issues/1190 - and a temporary fix is available

**Comment by tagrudev** [comment]
@DirkStevens using the form-data this.request.data is empty I don't see the fix in your issue

**Comment by DirkStevens** [comment]
@tagrudev there's a link to a temporary package that fixes (part of) the problem at #1190
